packages/editor/src/browser/editor.ts

```diff
@@ -150,6 +150,54 @@ export const enum EncodingMode {
     Decode
 }
 
+/**
+ * Options for searching matches in a model.
+ */
+export interface FindMatchesOptions {
+    /**
+     * The string used to search. If it is a regular expression, set `isRegex` to true.
+     */
+    searchString: string;
+    /**
+     * Limit the searching to only search inside the editable range of the model.
+     */
+    searchOnlyEditableRange: boolean;
+    /**
+     * Used to indicate the `searchString` is a regular expression.
+     */
+    isRegex: boolean;
+    /**
+     * Force the matching to match lower/upper case exactly.
+     */
+    matchCase: boolean;
+    /**
+     * Force the matching to match entire words only. Pass null otherwise.
+     */
+    wordSeparators: string | null;
```

Comment:
We should use `undefined` instead.

Reply:
Updated `wordSeparator` => boolean `matchWholeWord`